packages/openai-compatible/src/openai-compatible-chat-language-model.test.ts

```diff
@@ -646,32 +646,57 @@ describe('doGenerate', () => {
       expect(warnings).toEqual([]);
     });
 
-    it('should respect the includeUsage option', async () => {
```

Comment:
i noticed this was in the doGenerate describe block, so moved it below into correct place